assets/src/settings-page/analytics.js

```diff
@@ -0,0 +1,307 @@
+						} }
+						placeholder="{...}"
+						ref={ textAreaRef }
+						required
+						value={ config }
+					/>
+				</BaseControl>
+			</div>
+			<Button
+				isLink
+				onClick={ onDelete }
+				className="amp-analytics__delete-button"
+			>
+				<Icon icon={ trash } />
+				{ __( 'Remove entry', 'amp' ) }
+			</Button>
+		</PanelRow>
+	);
+}
+AnalyticsEntry.propTypes = {
+	config: PropTypes.string,
+	entryId: PropTypes.string,
+	onChange: PropTypes.func.isRequired,
+	onDelete: PropTypes.func.isRequired,
+	type: PropTypes.string,
+};
+
+/**
+ * Component handling addition and deletion of analytics entries.
+ */
+function AnalyticsOptions() {
+	const [ detailsInitialOpen, setDetailsInitialOpen ] = useState( null );
+
+	const getNewKey = useUniqueNewKey();
+	const { editedOptions, fetchingOptions, originalOptions, updateOptions } = useContext( Options );
+	const { analytics } = editedOptions;
+
+	/**
+	 * Set the initial open state of the details component.
+	 */
+	useEffect( () => {
+		if ( fetchingOptions ) {
+			return;
+		}
+
+		if ( 'boolean' === typeof detailsInitialOpen ) {
+			return;
+		}
+
+		setDetailsInitialOpen( ! Boolean( Object.keys( analytics ).length ) );
+	}, [ analytics, fetchingOptions, detailsInitialOpen ] );
+
+	return (
+		<div>
+
+			<details open={ ! Boolean( Object.keys( originalOptions.analytics ).length ) }>
+				<summary>
+					{ __( 'Learn about analytics for AMP.', 'amp' ) }
+				</summary>
+				<p dangerouslySetInnerHTML={
+					{ __html:
+						sprintf(
+							/* translators: 1: AMP Analytics docs URL. 2: AMP for WordPress analytics docs URL. 3: AMP analytics code reference. 4: amp-analytics, 5: {. 6: }. 7: <script>, 8: googleanalytics. 9: AMP analytics vendor docs URL. 10: UA-XXXXX-Y. */
+							__( 'For Google Analytics, please see <a href="%1$s" target="_blank">Adding Analytics to your AMP pages</a>; see also the <a href="%2$s" target="_blank">Analytics wiki page</a> and the AMP project\'s <a href="%3$s" target="_blank">%4$s documentation</a>. The analytics configuration supplied below must take the form of JSON objects, which begin with a %5$s and end with a %6$s. Do not include any HTML tags like %4$s or %7$s. A common entry would have the type %8$s (see <a href="%9$s" target="_blank">available vendors</a>) and a configuration that looks like the following (where %10$s is replaced with your own site\'s account number):', 'amp' ),
+							__( 'https://developers.google.com/analytics/devguides/collection/amp-analytics/', 'amp' ),
+							__( 'https://amp-wp.org/documentation/playbooks/analytics/', 'amp' ),
+							__( 'https://www.ampproject.org/docs/reference/components/amp-analytics', 'amp' ),
+							'<code>amp-analytics</code>',
+							'<code>{</code>',
+							'<code>}</code>',
+							'<code>&lt;script&gt;</code>',
+							'<code>googleanalytics</code>',
+							__( 'https://www.ampproject.org/docs/analytics/analytics-vendors', 'amp' ),
+							'<code>UA-XXXXX-Y</code>',
```

Comment:
URLs shouldn't be translated, unless I'm missing something.

```suggestion
							'https://developers.google.com/analytics/devguides/collection/amp-analytics/',
							'https://amp-wp.org/documentation/playbooks/analytics/',
							'https://www.ampproject.org/docs/reference/components/amp-analytics',
							'<code>amp-analytics</code>',
							'<code>{</code>',
							'<code>}</code>',
							'<code>&lt;script&gt;</code>',
							'<code>googleanalytics</code>',
							'https://www.ampproject.org/docs/analytics/analytics-vendors',
```

Reply:
This was copied and pasted from the former settings screen, so we'd better flag @westonruter on this before changing anything. I assume there was a reason for making the URLs translatable originally -- maybe for pointing to URLs with translated content.

Reply:
Right, URLs in WordPress core get translated to provide locale-specific URLs.

For sites in Spanish, this could point to: https://amp.dev/es/documentation/guides-and-tutorials/optimize-and-measure/configure-analytics/analytics-vendors/?format=websites